suite-native/device-manager/src/components/DeviceItem/DeviceItemIcon.tsx

```diff
@@ -0,0 +1,29 @@
+import { useSelector } from 'react-redux';
+
+import { Icon, IconSize } from '@suite-common/icons';
+import {
+    DeviceRootState,
+    PORTFOLIO_TRACKER_DEVICE_ID,
+    selectDeviceModelById,
+} from '@suite-common/wallet-core';
+import { TrezorDevice } from '@suite-common/suite-types';
+
+import { DeviceModelIcon } from '../DeviceModelIcon';
+
+type DeviceItemIconProps = {
+    deviceId: TrezorDevice['id'];
+    iconSize: IconSize;
+};
+
+export const DeviceItemIcon = ({ deviceId, iconSize }: DeviceItemIconProps) => {
```

Comment:
There used to be trezor icon and [according to the design](https://www.figma.com/file/mmDnYtctfjnreWhTjLfeQH/Passphrase?type=design&node-id=2382-11343&mode=design&t=pYULMai1iFohCY72-4) it should still be there, right?

<img width="383" alt="image" src="https://github.com/trezor/trezor-suite/assets/3729633/9be83af0-253d-4d6d-8af2-7885fb0db1c1">


vs 

<img width="597" alt="image" src="https://github.com/trezor/trezor-suite/assets/3729633/86f86f02-381d-4c3d-a372-f6703cbfb5d2">

Reply:
This is according to this design:
https://www.figma.com/file/mmDnYtctfjnreWhTjLfeQH/Passphrase?type=design&node-id=2325-66709&mode=design&t=DYCejprDAWSvtNQ4-0

Reply:
The icon should be there. i'll check where it went

Reply:
1db74c5c20

Reply:
Thanks for the icon fix, but the expanded state of device switcher is still not according to the design with no device 

This is how it looks in current state:
<img width="585" alt="image" src="https://github.com/trezor/trezor-suite/assets/3729633/1dd8c3c9-8274-4334-91a4-40ed5a8e2f6f">

Reply:
I probably dont understand what state is this. There is always a device. There is either portfolio device selected or a real device.

Reply:
@vytick try it on Android...

Reply:
Gotcha. I have updated the visual partially (trezor logo is even in opened switcher now, but this is where I can get. 

The design does not technically make a sense, since even in this very moment, portfolio tracker device is already selected, we just show it differently in the switcher because there are no accounts. This means that I dont know what should happen if user clicks on portfolio device there. Lets talk this tmrw.

<img width="300" alt="Screenshot 2024-05-14 at 22 38 46" src="https://github.com/trezor/trezor-suite/assets/2011829/7efa6558-993d-4e53-8d10-df08ba373f1a">